universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -19,14 +20,24 @@
 import services.program.ProgramDefinition;
 import services.question.QuestionDefinition;
 import views.BaseHtmlView;
+import views.StyleUtils;
 import views.Styles;
 import views.admin.AdminLayout;
 import views.components.FieldWithLabel;
+import views.components.Icons;
+import views.components.QuestionBank;
 
 public class ProgramBlockEditView extends BaseHtmlView {
 
   private final AdminLayout layout;
 
+  private static final String CREATE_BLOCK_FORM = "block-create";
+  private static final String DELETE_BLOCK_FORM = "block-delete";
+  private static final String UPDATE_BLOCK_FORM = "block-update";
+  private static final String ADD_QUESTION_FORM = "question-add";
+  private static final String DELETE_QUESTION_FORM = "question-delete";
```

Comment:
Suggestion (non-blocking): add `_ID` to the end of the constant name?

Reply:
Done.